cockatrice/src/carditem.cpp

```diff
@@ -317,7 +317,7 @@ void CardItem::mouseMoveEvent(QGraphicsSceneMouseEvent *event)
             2 * QApplication::startDragDistance())
             return;
         if (zone->getIsView()) {
-            const ZoneViewZone *const view = static_cast<const ZoneViewZone *const>(zone);
+            const ZoneViewZone *view = static_cast<const ZoneViewZone *>(zone);
```

Comment:
I removed the `const` from the pointer since the type of `zone` is non-const pointer. https://github.com/Cockatrice/Cockatrice/blob/89ab7f1f67aba119377312cc538d29c01662476c/cockatrice/src/carditem.h#L23 that should have been the issue.